recipes/msal/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "msal" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bf219535507cb4f777e8fdb60ef15413aa5006bab628a047617cae25b8d74127
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - requests >=2.0.0,<3
+    - pyjwt[crypto] >=1.0.0,<2
```

Comment:
You cannot use square brackets in dependendcies

Reply:
It looks weird to me as well, but that comes straight from the source package and it seems to work fine...

Reply:
What I am saying is that it won't work properly for conda. That is a method pip has for specifying optional dependendcies. It doesn't match the conda package spec, and so won't install those dependendcies correctly. It isn't possible to just copy-paste the dependendcy list from setup.py.

Reply:
Ok, I removed it. The pyjwt conda package seems to depend on `cryptography` anyway. The [crypto] bit just seems to be ignored by conda though. conda install pyjwt[crypto] is the same as conda install pyjwt.